tests/test_dictionaries_page.py

```diff
@@ -26,3 +39,44 @@ def test_public_activity_dict(app):
     assert dict_page.try_get_table_head_cell_i(1).text == u'Назва активності'
     assert dict_page.try_get_button_10 and dict_page.try_get_button_25 and dict_page.try_get_button_50 \
            and dict_page.try_get_button_100
+
+
+def test_public_activity_dict_2(app):
```

Comment:
test_public_activity_dict and test_public_activity_dict_2 are not the best names